pubspec.yaml

```diff
@@ -7,15 +7,15 @@ environment:
   sdk: ">=2.13.0 <3.0.0"
 
 dependencies:
-  analyzer: ^2.7.0
+  analyzer: ">=2.7.0 <6.0.0"
```

Comment:
6.0.0 is pulled from [here](https://github.com/dart-lang/test/blob/b82fc0b5c11fcfe4e636caef6b869f1a8cd4a803/pkgs/test/pubspec.yaml)